src/libsystemd-network/dhcp6-option.c

```diff
@@ -417,13 +417,13 @@ static int dhcp6_option_parse_pdprefix(DHCP6Option *option, DHCP6IA *ia,
         return 0;
 }
 
-int dhcp6_option_parse_ia(DHCP6Option *iaoption, DHCP6IA *ia) {
+int dhcp6_option_parse_ia(DHCP6Option *iaoption, DHCP6IA *ia, uint16_t *status_code) {
```

Comment:
hmm, we usually prefix our return parameters with `ret`. Hence maybe rename `status_code`→ `ret_status_code`